app/src/test/java/ar/scacchipa/jsonplaceholderviewer/data.kt

```diff
@@ -0,0 +1,78 @@
+package ar.scacchipa.jsonplaceholderviewer
+
+import ar.scacchipa.jsonplaceholderviewer.data.Comment
+import ar.scacchipa.jsonplaceholderviewer.data.Post
+
```

Comment:
maybe we can found a more descriptive name

Reply:
The name was improved